build.sbt

```diff
@@ -186,14 +186,15 @@ val commonSettings = Seq(
   tlFatalWarningsInCi := false,
   tlJdkRelease := Some(8),
   tlSkipIrrelevantScalas := true,
+  scalacOptions ~= { _.filterNot(_ == "-source:3.0-migration") },
   scalacOptions ++= {
     CrossVersion.partialVersion(scalaVersion.value) match {
       case Some((3, _)) =>
         Seq(
           // required by magnolia for accessing default values
-          "-Xretain-trees",
+          "-Yretain-trees",
           // tolerate some nested macro expansion
-          "-Ymax-inlines",
+          "-Xmax-inlines",
```

Comment:
Is it related to new Magnolia version?

Reply:
yes, those are required by magnolia with scala 3. Some macros are generating too long inlines.